internal/controller/supervisorconfig/activedirectoryupstreamwatcher/active_directory_upstream_watcher.go

```diff
@@ -387,7 +387,8 @@ func (c *activeDirectoryWatcherController) updateStatus(ctx context.Context, ups
 	}
 }
 
-func microsoftUUIDFromBinaryAttr(attributeName string) func(entry *ldap.Entry) (string, error) {
+//nolint:gochecknoglobals // this needs to be a global variable so that tests can check pointer equality
+var microsoftUUIDFromBinaryAttr = func(attributeName string) func(*ldap.Entry) (string, error) {
```

Comment:
Would be nice if golang allowed `const` for a global func.

Reply:
I agree. I wish `const` was enabled for more things (slices, maps) etc.